src/test/java/NewPlaylistTestsUsingPOM.java

```diff
@@ -0,0 +1,22 @@
+import POM.BasePage;
+import POM.HomePage;
+import POM.LoginPage;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class NewPlaylistTestsUsingPOM extends BaseTest {
+    private String notificationMessage = "Created playlist \"Iulia's Playlist1.\"";
+
+    @Test
+    public void createANewPlaylist() {
+        LoginPage loginPage = new LoginPage(driver);
+        HomePage homePage = new HomePage(driver);
+        BasePage basePage = new BasePage(driver);
+        loginPage.login();
+        homePage.createNewPlaylist();
```

Comment:
![image](https://user-images.githubusercontent.com/104761482/200290444-883e71da-c185-4460-b593-cfd4b6cff09d.png)

if your problem is still not solved, you need a DOM-level solution. So, we need an interface that helps to execute JavaScript with Selenium WebDriver. It is called JavaScriptExecutor. JavaScript works at the DOM level. Therefore, we are able to click any web element by using JavaScript executor.

For, example

```
        ((JavascriptExecutor) driver).executeScript("arguments[0].scrollIntoView(true);", (createANewPlaylistElement));
        Thread.sleep(2000);
        ((JavascriptExecutor) driver).executeScript("arguments[0].click();", (createANewPlaylistElement));
```

Clicking Elements in Selenium using JavaScript (https://www.baeldung.com/java-selenium-javascript)

![image](https://user-images.githubusercontent.com/104761482/200290910-0aae0b05-60ab-4c4e-8252-a4b1538a5bd1.png)